### PR TITLE
Amanda's Winter Code Changes

### PR DESCRIPTION
**Pull request by picoHacking**
Before issuing a pull request, please see the contributing page.
All my lift/intake/odometry programming done over winter break so far.

### CODE REVIEW

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/hardware/Drivetrain.java [hunk 1]
```diff
@@ -43,6 +43,16 @@ public Drivetrain(DcMotor front_left, DcMotor front_right, DcMotor back_left, Dc
...
+
+        front_right.setMode(DcMotor.RunMode.STOP_AND_RESET_ENCODER);
+        back_right.setMode(DcMotor.RunMode.STOP_AND_RESET_ENCODER);
+        front_left.setMode(DcMotor.RunMode.STOP_AND_RESET_ENCODER);
+        back_left.setMode(DcMotor.RunMode.STOP_AND_RESET_ENCODER);
+
+        front_right.setMode(DcMotor.RunMode.RUN_USING_ENCODER);
+        back_right.setMode(DcMotor.RunMode.RUN_USING_ENCODER);
+        front_left.setMode(DcMotor.RunMode.RUN_USING_ENCODER);
+        back_left.setMode(DcMotor.RunMode.RUN_USING_ENCODER);
```
Comment (Siddapa) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/hardware/Drivetrain.java:55]: Is the intention to use the inbuilt RUN_TO_POSITION?

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/util/Status.java [hunk 2]
```diff
@@ -41,12 +42,14 @@
     }};
     public static final double BUCKET_WAIT_TIME = 0.2;
     public static final double PITSTOP_WAIT_TIME = 0.15;
+    public static final double OUTRIGGER_UP = 1.0;
+    public static final double OUTRIGGER_DOWN = 0.0;
```
Comment (Siddapa) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/util/Status.java:53]: Refactor into Hashmap to keep consistent with other sub-assemblies

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/hardware/Drivetrain.java (hunk 1 above)
Comment (picoHacking) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/hardware/Drivetrain.java:55]: No. I just wanted to access the encoders. Apparently I didn't need the run mode.

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/util/Status.java (hunk 2 above)
Comment (picoHacking) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/util/Status.java:53]: Okay, I will do that.

**Pull request merged**